corehq/pillows/sms.py

```diff
@@ -58,18 +57,16 @@ def calc_meta(cls):
         return cls.calc_mapping_hash({"es_meta": cls.es_meta, "mapping": cls.default_mapping})
 
     def change_transport(self, doc_dict):
-        domain = doc_dict.get('domain')
-        if domain and SQL_SMS_PILLOW.enabled(domain):
-            return
-
-        super(SMSPillow, self).change_transport(doc_dict)
+        # SMS changes don't go to couch anymore. Let the SqlSMSPillow process
+        # changes from now on.
+        # Also, we explicitly need this to be a no-op because we're going to
+        # delete all sms from couch and don't want them to be deleted from
+        # elasticsearch.
+        return
```

Comment:
I think you can keep it for now. It's on my backlog to sort this out at which point we can remove this class entirely. I imagine I'll start with this one since most of the work is already done, so hopefully will get to that next week.